README.md

```diff
@@ -16,51 +16,99 @@ These OpenXR samples are using C++17 and D3D11. The same source code works acros
+Create one `xrSwapchain` for both left and right eye using `arraySize=2` for color swapchain, and one for depth.
+Render the left eye into slice 0 and the right eye into slice 1.
+Use a shader with VPRT and instanced draw calls for stereoscopic rendering to minimize GPU load.
+This also enables the runtime's optimization to achieve the best performance on HoloLens 2.
+Alternatives to using a texture array, such as double-wide rendering or a separate swapchain per eye, will result in runtime post-processing which comes at a significant performance penalty.
 
 ## Render with recommended rendering parameters and frame timing
 
-Always render with the recommended view configuration width/height, and always use `xrLocateViews` API to query for the recommended view pose, fov, and other rendering parameters before rendering. Always use the `XrFrameEndInfo.predictedDisplayTime` from the latest `xrWaitFrame` call when querying for poses and views. This allow HoloLens to adjust rendering and optimize visual quality for the person who is wearing the HoloLens.
+Always render with the recommended view configuration width/height (`recommendedImageRectWidth` and `recommendedImageRectHeight` from `XrViewConfigurationView`), and always use `xrLocateViews` API to query for the recommended view pose, fov, and other rendering parameters before rendering.
+Always use the `XrFrameEndInfo.predictedDisplayTime` from the latest `xrWaitFrame` call when querying for poses and views.
+This allow HoloLens to adjust rendering and optimize visual quality for the person who is wearing the HoloLens.
 
 ## Submit depth buffer for projection layers
 
-Always use `XR_KHR_composition_layer_depth_extension` and submit the depth buffer together with the projection layer when submitting a frame to `xrEndFrame`.   This can help hologram stability by enabling the hardware depth reprojection on HoloLens 2.
+Always use `XR_KHR_composition_layer_depth_extension` and submit the depth buffer together with the projection layer when submitting a frame to `xrEndFrame`.
+This can help hologram stability by enabling the hardware depth reprojection on HoloLens 2.
 
 ## Choose a reasonable depth range
 
-Prefer a narrower depth range to scope the virtual content to help hologram stability on HoloLens. For example, the OpenXrProgram.cpp sample is using 0.1 to 20 meters. Use [reversed-Z](https://developer.nvidia.com/content/depth-precision-visualized) for a more uniformed depth resolution.  Note, on HoloLens 2, using the preferred `DXGI_FORMAT_D16_UNORM` depth format can help achieve better frame rate and performance.  Therefore above practice for depth range is more important.
+Prefer a narrower depth range to scope the virtual content to help hologram stability on HoloLens.
+For example, the OpenXrProgram.cpp sample is using 0.1 to 20 meters.
+Use [reversed-Z](https://developer.nvidia.com/content/depth-precision-visualized) for a more uniformed depth resolution.

```

Comment:
I would tweak this to "uniform depth resolution" and mention that reversed-Z becomes especially important when using a 16-bit depth buffer, as suggested above.

Reply:
Or also below :)

Perhaps move the `D16_UNORM` bullet one higher, and then motivate reversed-Z based on that choice, since your depth resolution starts out that much lower.